api/management/commands/loaddata.py

```diff
@@ -0,0 +1,17 @@
+import os
+import csv
```

Comment:
Style https://www.python.org/dev/peps/pep-0008/#imports:

Imports should be grouped in the following order:

1. standard library imports
+ related third party imports
+ local application/library specific imports

You should put a blank line between each group of imports.